android/source/VideoLocker/build.gradle

```diff
@@ -232,7 +232,7 @@ android {
 
             versionCode 40
             versionName '1.0.03'
-            minSdkVersion 16
+            minSdkVersion 14
             targetSdkVersion 21
```

Comment:
@shahidtamboli  Please use extra properties (defined in the same file) for minSdkVersion, targetSdkVersion, versionCode and versionName. 
Otherwise, Looks good to me.